test_main.py

```
@@ -238,8 +238,79 @@ def test_export_unlabeled_nodes(self, falkordb_connection, temp_dir):
+        export_graph(test_graph, "localhost", 6379)
+        
+        # Should create separate files per label/type (default behavior)
+        assert os.path.exists("nodes_Person.csv"), "Should create nodes_Person.csv by default"
+        assert os.path.exists("nodes_Company.csv"), "Should create nodes_Company.csv by default"
+        assert os.path.exists("edges_WORKS_FOR.csv"), "Should create edges_WORKS_FOR.csv by default"
+        
+        # Should NOT create combined files
+        assert not os.path.exists("nodes.csv"), "Should not create nodes.csv in split mode"
+        assert not os.path.exists("edges.csv"), "Should not create edges.csv in split mode"
+    
+    def test_export_pagination_large_graph(self, falkordb_connection, temp_dir):
```

Ruff (0.14.2)

Unused method argument: `temp_dir`

(ARG002)